src/Middleware/RateLimiting/src/RateLimitingOptions.cs

```diff
@@ -0,0 +1,53 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Threading.RateLimiting;
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.RateLimiting;
+
+/// <summary>
+/// Specifies options for the <see cref="RateLimitingMiddleware"/>.
```

Comment:
The type is non-public so I imagine the cref won't resolve. Perhaps

```suggestion
/// Specifies options for the rate limiting middleware.
```